### PR TITLE
Feature 6

### PR DESCRIPTION
**Pull request by Jay-chand-yadav-Fexle-May2022**

### CODE REVIEW

File: src/classes/AccountTriggerHandlerTests.cls [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    
+    @isTest
+    private static void updateTheAccountsNumberTest() {
+    	
+        List<Account> listOfAccounts = new List<Account>();
+        listOfAccounts.add(new Account(Name = 'Account4', Type = 'Saving',Email__c = 'testing3@fexle.com'));
+        listOfAccounts.add(new Account(Name = 'Account5', Type = 'FD',Email__c = 'testing4@fexle.com'));
+        
+        Test.startTest();
+        	insert listOfAccounts;
```
Comment (Jaychandyadav) [inline src/classes/AccountTriggerHandlerTests.cls]: Use one tab

File: src/classes/AccountTriggerHandlerTests.cls [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    
+    @isTest
+    private static void sendEmailTest() {
+        
+        List<Account> listOfAccounts_1 = new List<Account>();
+        listOfAccounts_1.add(new Account(Name = 'Account6', Type = 'Saving', Email__c = 'testing5@test.com'));
+        listOfAccounts_1.add(new Account(Name = 'Account7', Type = 'FD', Email__c = 'testing6@test.com'));
+        
+        Test.startTest();
+        	insert listOfAccounts_1;
```
Comment (Jaychandyadav) [inline src/classes/AccountTriggerHandlerTests.cls]: Use single tab

File: src/classes/AccountTriggerHandlerTests.cls (hunk 1 above)
Comment (Jaychandyadav) [inline src/classes/AccountTriggerHandlerTests.cls]: Use one tab

Comment (Jay-chand-yadav-Fexle-May2022) [reply src/classes/AccountTriggerHandlerTests.cls]: Ok

**Review by Jaychandyadav — APPROVED** [review]

**Pull request merged**